index.html

```diff
@@ -6,8 +6,67 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/7cbe454c49.js" crossorigin="anonymous"></script>
 </head>
 <body>
+    <header class="header-container">
+    <div class="sub-container">
+        <a href="#" class="logo-cont">Olamarx</a>
+      <button type="submit" class="header-font"><i class="fa fas-solid fa-bars font"></i> </button> 
+    </div>
+    </header>
+
+    <section>
+        <div class="main-container">
+            <div class="images">
+
+                <div class="img left-img">
+                    <img src="./images/Rectangle6.png" alt="Rectangle">
+                    <img src="./images/Ellipse11.png" alt="Ellipse">
+                </div>
+
+                <div class="img center-img">
+                    <img src="./images/Rectangle7.png" alt="Rectangle">
+                    <img src="./images/Ellipse5.png" alt="Ellipse">
+                </div>
+
+                <div class="img rigth-img">
+                    <img src="./images/Rectangle4.png" alt="Rectangle">
+                    <img src="./images/Ellipse3.png" alt="Ellipse">
+                </div>
+
+            </div>
+        </div>
+<div class="porfolio-content">
+    <div class="portfolio-h1">
+        <h1>I'm Olamarx </h1>
+        <h1 class="span-portfolio-h1-1">Glad to have</h1>
+        <h1 class="span-portfolio-h1-2">you here!</h1> 
+    </div>
+<div class="portfolio-content-para">
+    <p class="portfolio-para">
+        I am a Web Developer! I can help you build a standard and responsive website. 
+        You can look through some of my handiworks and experience. <span class="portfolio-para-span">Do you like what you see?</span>
+   LET US CONNECT </p>
+</div>
+</div>
+
+
+<p class="connect-with-me">
+        CONNECT WITH ME
+</p>
+
+<div class="contact-link">
+    <div class="social-link">  
+    <a href="#"><i class="fa fas-brands fa-twitter social"></i></a>
+    <a href="#"><i class="fa fas-brands fa-linkedin social"></i></a>
+    <a href="#"><i class="fa fas-brands fa-facebook social"></i></a>
+    <a href="#"><i class="fa fas-brands fa-github social"></i></a>
+</div>
```

Comment:
- [ ] Please kindly ensure the list of icons is grouped with a `ul` tag, and `li` per icon; each icon has a link (`a` tag)
- [ ] Please ensure to use the list of icons specified in the original design. You can export from figma and then use in your project